__init__.py

```diff
@@ -18,14 +23,20 @@ def runtime_requirements(self):
         )
 
     def initialize(self):
+        self.settings.merge(DEFAULT_SETTINGS, new_only=True)
+        self.unconfigured_message = f"ChatGPT not configured yet, please set your API key in {self.settings_path}"
```

Comment:
any specific reason this is a variable? we dont usually do that for logs so curious what you had in mind

Reply:
The biggest reason is that it's logged in two places, so it made sense to make it a variable. Especially since it's so long

Reply:
I only see it once in in line 96 ?

Reply:
Ah right, I just deleted one 🙂